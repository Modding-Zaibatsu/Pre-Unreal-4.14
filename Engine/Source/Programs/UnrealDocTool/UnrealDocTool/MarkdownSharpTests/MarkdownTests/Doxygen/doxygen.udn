%doxygen:FStreamingManagerTexture::FPendingPrimitiveType%

| test table header A | test table header B |  
| ------- | ------------|  
| test table cell A | %doxygen:FStreamingManagerTexture::FPendingPrimitiveType% |  

%doxygen:AActor::GetInstigator() const%

%doxygen:AActor::GetInstigator<class T>() const%