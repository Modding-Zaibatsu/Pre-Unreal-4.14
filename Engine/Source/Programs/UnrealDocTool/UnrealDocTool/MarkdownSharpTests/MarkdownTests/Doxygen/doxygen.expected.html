<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" dir="ltr" lang="en-US">
    <head>
    </head>
    <body>
        <p>First sentence. The rest of the comment.</p>
        
        <table>
            <caption></caption>
            <col  align="" />
            <col  align="" />
            <thead>
                <tr>
                    <th rowspan="1" colspan="1">
                        test table header A 
                    </th>
                    <th rowspan="1" colspan="1">
                        test table header B 
                    </th>
                </tr>
            </thead>
            <tbody>
                <tr>
                    <td rowspan="1" colspan="1" align="">
                        test table cell A
                    </td>
                    <td rowspan="1" colspan="1" align="">
                        <p>First sentence. The rest of the comment.</p>
                    </td>
                </tr>
            </tbody>
        </table>
        
        <p>Returns the instigator for this actor, or NULL if there is none.</p>
        
        <p>Get the instigator, given a specific class.</p>
    </body>
</html>
